### ISSUE TITLE
[Suggestion] Add a way to hide warnings and errors from certain mods in a list.

### ISSUE THREAD

**Issue by Corvus20** [rimpy-custom/RimPy#88]
There are some mods that still work despite not matching the game version, such as the ones shown in the screenshot below. If you could add some sort of "Ignore warnings and errors" option under the advanced operations, it would be a very welcome addition.

![image](https://user-images.githubusercontent.com/75586106/189011798-eb9a94c4-8191-41e7-9bf0-8d025462e883.png)

**Comment by Scorpiopt** [comment]
agree with this sugestion

**Comment by MsVella** [comment]
Agreed with this, i get a bit annoyed constantly seeing and being stopped by errors for texture packs that are a version out of date or have a mod that been continued under another package ID
